app/code/Magento/Email/Model/AbstractTemplate.php

```diff
@@ -534,10 +534,9 @@ protected function cancelDesignConfig()
      */
     public function setForcedArea($templateId)
     {
-        if ($this->area) {
-            throw new \LogicException(__('Area is already set'));
```

Comment:
Was there any logic in this exception before? Any idea why it was introduced?

Reply:
Check the blame view of the file in the original repo and you see that the code was never really touched ;-)

Reply:
@DanielRuf Is there any patch available for temporary. I need to fix this in my Magento v.2.2.4.

Reply:
@hitesh-wagento simply add `.patch` to the pull/number URL and you get https://patch-diff.githubusercontent.com/raw/magento/magento2/pull/15137.patch